tools/deploy.sh

```diff
@@ -154,7 +154,7 @@ if [ "${DEPLOY_TLS}" == "true" ]; then
     fi
 
     if [ "${DEPLOY_BMO}" == "true" ]; then
-        cp "${IRONIC_CACERT_FILE}" "${SCRIPTDIR}/deploy/tls/ca.crt"
+        cp "${IRONIC_CACERT_FILE}" "${SCRIPTDIR}/config/tls/ca.crt"
```

Comment:
Looks good. while we are fixing this script, can you please change line 54 this way :
```
sudo chown -R "${USER}:$(id -gn)" "${IRONIC_DATA_DIR}"
```

Reply:
Done